index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200;500;600;700&display=swap" rel="stylesheet">
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <title>Document</title>
+</head>
+<body>
+     <section class="basic-section">
+      <header class="header">
+       <nav class="nav">
+        <div class="logo">
+            <h3>KMarketer</h3>
+        </div>
+            <ul class="nav_bar">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Product</a></li>
+                <li><a href="#">Pricing</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a><img alt="svg" src="img/Vector.png"></a></li>
+                <li><a><img alt="svg" src="img/icnshopping-cart.icn-xs.png"></a></li>
+             </ul>
+        </nav>
+    </header>
+
+         <div class="main_content">
+            <p class="maintext">We Take Care of<br>      the Future</p>
+            <p class="maintext2">We know how large objects will act, but things on a <br> 
+             small scale just do not act that way.</p>
+        <div class="btn">
+             <a href="#" class="buttonquote">   Get Quote Now   </a>
```

Comment:
Please delete spaces <a href="#" class="buttonquote">   Get Quote Now   </a> and in line 36